R/opart_gaussian.R

```diff
@@ -37,16 +37,15 @@ opart_gaussian <- structure(function(data, penalty) {
     stop("penalty must be a finite numeric value")
   }
 
-
   result <- .C("opart_gaussian_interface",
-               as.integer(length(data)),
-               as.double(data),
-               as.double(penalty),
+               n_data = as.integer(length(data)),
+               data_ptr = as.double(data),
```

Comment:
in R code we do not usually have pointers so using `data_ptr` is a bit confusing. i suggest changing to `data.vec` which suggests it is a vector.